source/Helpers/EventsHandler.cs

```diff
@@ -42,6 +40,26 @@ private static void GlobalHookKeyPressHandler(object sender, KeyEventArgs e)
             }
         }
 
+        private static void GlobalHookMouseScrollHandler(object sender, MouseEventArgs e)
+        {
+            if (!TradeClient.IsReady || !ProcessHelper.IsPathOfExileInFocus())
+            {
+                return;
+            }
+
+            // CTRL + Scroll to move between stash tabs while not hovering stash
+            int windowWidth = ProcessHelper.GetActiveWindowWidth();
+            if (e.X > 0.35 * windowWidth)
```

Comment:
Shouldn’t you verify here that stash tab is currently open? So, you this function will not work in fight.

Reply:
Is there a way to know that? Scanning pixels?

Reply:
I don't think it's a problem the way it works for now. We we see one, we can create an issue.

Reply:
Yeah, I think, this is the only way to achieve it - to get some pixel colors. But it is not so simple because of the different screen resolutions. Feels like it is out of the main program functionality.

Reply:
I think Sidekick could eventually do some data fetching using pixel scanning in the future.